teamcity/assets/service_checks.json

```diff
@@ -1 +1,75 @@
-[]
+[
+    {
+        "agent_version": "7.40.0",
```

Comment:
Can you indicate on the service check descriptions if they are submitted by the rest or openmetrics instances?